FirebaseCore/Sources/Private/FIRAppInternal.h

```diff
@@ -14,10 +14,12 @@
  * limitations under the License.
  */
 
-#if __cplusplus
-#import <FirebaseCore/FIRApp.h>
-#else
+#if SWIFT_PACKAGE
 #import "FIRApp.h"
+#else
+// Above is not findable in some CocoaPods scenarios, including Objective C++
```

Comment:
This comment is talking about CocoaPods, C++, and Firebase versions, but the `#if` is referring to `SWIFT_PACKAGE`. Since It seems like SPM is the exception and we'd be interested in just finding the single common way to import things, maybe it be worth discussing why SPM can't use the module-qualified form? 

In its current form, I don't really understand this comment and it doesn't really help me get context. In which CocoaPods builds does this not work? If it's at all complicated, it's probably worth spelling this out in detail in your headers/imports doc and just linking here.

Also this is phrased in a way that seems to indicate all three of these conditions are required when I think it's any of them, right? At the very least change "and using" to "or using", but also consider splitting this into bullets and giving more details on what's wrong in each case.

Finally, what's special about FirebaseCore 6.27.0? Did it change something? It may be worth describing what changed about it that makes this version special.

Reply:
Removed confusing comment in favor of a TODO when I get back to Swift PM.  For now, these three headers are unchanged for CocoaPods.